### PR TITLE
Documentation: improve instructions for Windows cross compiling

### PR DESCRIPTION
**Pull request by ellishg**
I wanted to help clarify and improve the instructions for cross compiling for windows. On Linux, the cmake options were not correct so I added the correct options. Also, the cmake options should be comma separated with no spaces between options.

### CODE REVIEW

**Comment by sparkasaurusRex** [comment]
@compnerd Who else should I tag?

File: docs/WindowsCrossCompile.md [hunk 1]
```diff
@@ -32,13 +32,34 @@ build the static libraries. Note that cross-compiling will require the use of
...
+-DCMAKE_RANLIB=<path to llvm-ranlib>,\
+-DSWIFT_SDKS='LINUX;WINDOWS',\
+-DSWIFT_WINDOWS_x86_64_ICU_I18N_INCLUDE=<path to ICU i18n includes>,\
+-DSWIFT_WINDOWS_x86_64_ICU_UC_INCLUDE=<path to ICU UC includes>,\
+-DSWIFT_WINDOWS_x86_64_ICU_I18N=<path to ICU i18n lib>,\
+-DSWIFT_WINDOWS_x86_64_ICU_UC=<path to ICU UC lib>
+```
+
+## 4. Build the Windows Standard Libraries with `ninja`
+From the build directory we can build the windows standard libraries using
```
Comment (xwu) [inline docs/WindowsCrossCompile.md]: Please capitalize Windows. Also, this is known as the _Swift_ runtime and standard library (for Windows), not the "Windows standard libraries." (And "standard library" isn't capitalized for Swift.)

File: docs/WindowsCrossCompile.md [hunk 2]
```diff
@@ -32,13 +32,34 @@ build the static libraries. Note that cross-compiling will require the use of
...
-  -DSWIFT_WINDOWS_ICU_I18N=<path to ICU i18n lib>, \
-  -DSWIFT_WINDOWS_ICU_UC=<path to ICU UC lib>
+--extra-cmake-options=-DSWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER=FALSE,\
+-DCMAKE_AR=<path to llvm-ar>,\
+-DCMAKE_RANLIB=<path to llvm-ranlib>,\
+-DSWIFT_SDKS=WINDOWS,\
+-DSWIFT_WINDOWS_ICU_I18N_INCLUDE=<path to ICU i18n includes>,\
+-DSWIFT_WINDOWS_ICU_UC_INCLUDE=<path to ICU UC includes>,\
+-DSWIFT_WINDOWS_ICU_I18N=<path to ICU i18n lib>,\
+-DSWIFT_WINDOWS_ICU_UC=<path to ICU UC lib>
```
Comment (compnerd) [inline docs/WindowsCrossCompile.md]: These should also use the ARCH specific variables.

**Review by compnerd — CHANGES_REQUESTED** [review]

**Comment by compnerd** [comment]
@sparkasaurusRex - for future changes, I think that these could've been collapsed into a single commit.

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Review by compnerd — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Pull request merged**